topydo.conf

```diff
@@ -6,6 +6,7 @@ default_command             = ls
 ; archive_filename          = done.txt
 colors                      = 1
 identifiers                 = linenumber ; or: text
+; backup                    = 1
```

Comment:
Or on second thought, make it just a boolean and eliminate the `backup_file` option. And make the location always relative to the `todo.txt` file (same directory, but something like `.filename.bak`, where filename is the value of the `filename` option.

The motivation is to prevent that a changeset contains revisions of other todo.txt files than the currently configured one. Having a different revision on top of the undo stack would break the `undo` command.